### PR TITLE
Cart View styling and Navbar update

### PR DESCRIPTION
**Pull request by Laszlo-JFLMTCO**
@njgheorghita @Dpalazzari @drod1000 

Here is the latest pull request for Cart View styling. In addition, other improvements covered by this pull request:  
- Updated Navbar with buttons to improve site navigation,  
- Temporarily redirect site ROOT to Outings view,  
- Eliminate deprecated `should` from Cart feature testing (thanks @drod1000 for the correct syntax!)

Already includes my local merge with latest Master and all tests pass!

### CODE REVIEW

File: spec/features/visitor_views_cart_spec.rb [hunk 1]
```diff
@@ -24,13 +24,13 @@
       expect(page).to have_content(first.title)
       expect(page).to have_content(first.description)
       expect(page).to have_content(first.base_cost * politician.multiplier)
-      page.find('#outing-photo')['src'].should have_content(first.image_url)
+      expect(page.find('#outing-photo')['src']).to have_content(first.image_url)
     end
```
Comment (njgheorghita) [inline spec/features/visitor_views_cart_spec.rb:28]: nice find @drod1000 @Laszlo-JFLMTCO

**Comment by njgheorghita** [comment]
looks good to me!

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  
+  root :to => 'outings#index'
```
Comment (Dpalazzari) [inline config/routes.rb:3]: Is the colon in the right place here? @Laszlo-JFLMTCO

Comment (Dpalazzari) [reply config/routes.rb:3]: root to: 'outings#index'

**Comment by drod1000** [comment]
Looks good, merging now!

**Pull request merged**